content/games.html

```diff
@@ -78,7 +78,7 @@
           rowClass = 'greyout';
 
         var row = $('<tr class="'+rowClass+'">');
-        row.append('<td>'+game['name'].replace(/\\\'/g, "'")+'<br /><span class="gameSubline">'+game['address']+'</span></td>');
+        row.append('<td>'+game['name'].replace(/\\\'/g, "'")+'<br /><span class="gameSubline"><a href="openra://'+game['address']+'">'+game['address']+'</a></span></td>');
```

Comment:
Should have mentioned this earlier, but we should also hide the links for in progress games (since joining will never work).